### PR TITLE
DOCS-#5292: Make readme image links raw so they render on pypi.org.

### LINKED ISSUE #5292: DOCS: README images don't render on modin pypi page
e.g. see the placeholder image [here](https://pypi.org/project/modin/) where the gif should be, shown at end of this post. 
Seems we need "raw" URLs as specified [this answer](https://stackoverflow.com/a/46875147/17554722) or [this](https://stackoverflow.com/a/65945193/17554722).

Screen Shot 2022-11-30 at 2 11 57 PM

### PR DESCRIPTION
**Pull request by mvashishtha**
Signed-off-by: mvashishtha <mahesh@ponder.io>

## What do these changes do?

I got all the raw URLS with "copy image address" on the images shown in my browser.

- [x] first commit message and PR title follow format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
  > **_NOTE:_**  If you edit the PR title to match this format, you need to add another commit (even if it's empty) or amend your last commit for the CI job that checks the PR title to pick up the new PR title.
- [x] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] signed commit with `git commit -s` 
- [x] Resolves #5292
- [x] tests added and passing
- [x] module layout described at `docs/development/architecture.rst` is up-to-date

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ scalability in a cluster.
 We designed [Modin's architecture](https://modin.readthedocs.io/en/latest/development/architecture.html)
 to be modular so we can plug in different components as they develop and improve:
 
-<img src="docs/img/modin_architecture.png" alt="Modin's architecture" width="75%"></img>
+<img src="https://github.com/modin-project/modin/raw/master/docs/img/modin_architecture.png" alt="Modin's architecture" width="75%"></img>
```
Comment (mvashishtha) [inline README.md:197]: diff looks wrong in the markdown preview for this PR, but architecture image looks fine on my branch: https://github.com/mvashishtha/modin/tree/5292-fix-make-readme-image-links-raw

File: README.md [hunk 2]
```diff
@@ -26,28 +26,28 @@ cores. Modin works especially well on larger datasets, where pandas becomes pain
 
 By simply replacing the import statement, Modin offers users effortless speed and scale for their pandas workflows:
 
-<img src="https://github.com/modin-project/modin/blob/master/docs/img/Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
+<img src="https://github.com/modin-project/modin/raw/master/docs/img/Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
```
Comment (vnlitvinov) [inline README.md:29]: if we're giving fixed links, I would rather we specify fully static link (with commit hash, like in the first line)

here and in other places

Comment (mvashishtha) [reply README.md]: Done.

**Review by vnlitvinov — APPROVED** [review]
LGTM!

**Pull request merged**